components/network/ssh-askpass/network-ssh-askpass.p5m

```diff
@@ -0,0 +1,35 @@
+#
```

Comment:
Do we really want two ssh-askpass packages?
Other issue: please, use new CDDL header in your files.